docs/command_reference.md

```diff
@@ -0,0 +1,235 @@
+
+## Initializing a trusted collection
+
+Notary can initialize a trusted collection with the `notary init` command:
+```bash
+$ notary init <GUN>
+```
+
+This command will generate targets and snapshot keys locally for the trusted collection, and try to locate a root key to use in the specified metadata cache (from `-d` or the config).
+If notary cannot find a root key, it will generate one.  For all keys, notary will also prompt for a passphrase to encrypt the private key material at rest.
+
+If you'd like to initialize your trusted collection with a specific root key, there is a flag:
+```bash
+$ notary init <GUN> --rootkey <key_file>
+```
+
+Note that you will have to run a publish after this command, as there are staged changes to initialize the trusted collection that have not been pushed to a ntoary server:
+```bash
+$ notary publish <GUN>
+```
+
+## Auto-publish changes
+
+Instead of manually running `notary publish` after each command, you can use the `-p` flag to auto-publish the changes from that command.
+For example:
+
+```bash
+$ notary init -p <GUN>
+```
+
+## Manage staged changes
+
+The Notary CLI client stages changes before publishing them to the server.
+You can manage the changes that are staged by running:
+
+```bash
+# Check what changes are staged
+$ notary status <GUN>
+
+# Unstage a specific change
+$ notary status <GUN> --unstage 0
+
+# Alternatively, unstage all changes
+$ notary status <GUN> --reset
+```
+
+Note that `<GUN>` can take on arbitrary values, but when working with Docker Content Trust they are structured like `<url>/<account>/<repository>`.  For example `docker.io/library/alpine` for the [Docker Library Alpine image](https://hub.docker.com/r/library/alpine).
+
+When you're ready to publish your changes to the Notary server, run:
+
+```bash
+$ notary publish <GUN>
+```
+
+
+## Delete trust data
+
+Users can remove all notary signed data for a trusted collection by running:
+
+```bash
+$ notary delete <GUN> --remote
+```
+
+If you don't include the `--remote` flag, Notary deletes local cached content
+but will not delete data from the Notary server.
+
+## Change the passphrase for a key
+
+The Notary CLI client manages the keys used to sign the trusted collection. These keys are encrypted at rest.
+To list all the keys managed by the Notary CLI client, run:
+
+```bash
+$ notary key list
+```
+
+To change the passphrase used to encrypt one of the keys, run:
+
+```bash
+$ notary key passwd <key_id>
+```
+
+## Rotate keys
+
+If one of the private keys is compromised you can rotate that key, so that
+content that was signed with those keys stop being trusted.
+
+For keys that are kept offline and managed by the Notary CLI client, such the
+keys with the root, targets, and snapshot roles, you can rotate them with:
+
+```bash
+$ notary key rotate <GUN> <key_role>
+```
+
+The Notary CLI client generates a new key for the role you specified, and
```

Comment:
Would it make sense in this section to mention that `key rotate` replaces all existing keys on that role?